policyengine-client/src/countries/us/us.jsx

```diff
@@ -278,6 +297,8 @@ export class US extends Country {
         "ca_cvrp_vehicle_rebate_amount",
         "qualified_tuition_expenses",
         "is_eligible_for_american_opportunity_credit",
+        // Tax unit.
+        "premium_tax_credit",
```

Comment:
This avoids calculation of the ACA premium tax credit, which is volatile and which we don't want to show without Medicaid.